python/ray/data/_internal/planner/planner.py

```diff
@@ -38,6 +40,9 @@ def _plan(self, logical_op: LogicalOperator) -> PhysicalOperator:
         if isinstance(logical_op, Read):
             assert not physical_children
             physical_op = _plan_read_op(logical_op)
+        elif isinstance(logical_op, Write):
+            assert len(physical_children) == 1
+            physical_op = _plan_write_op(logical_op, physical_children[0])
         elif isinstance(logical_op, AbstractUDFMap):
```

Comment:
```suggestion
        elif isinstance(logical_op, AbstractMap):
```

Can we do this instead?

Reply:
We cannot now. The `Read` is also an AbstractMap, and it has no input.

Reply:
Ok that's a bit weird, I thought the inputs would be the read tasks. Otherwise it's not really a Map...